src/platform/forms-system/test/js/containers/FormPage.unit.spec.jsx

```diff
@@ -150,7 +160,61 @@ describe('Schemaform <FormPage>', () => {
 
     tree.getMountedInstance().goBack();
 
-    expect(router.push.calledWith('first-page'));
+    expect(router.push.calledWith('/first-page')).to.be.true;
+  });
+  it('should go to the custom path passed to the goBack function', () => {
+    const router = {
+      push: sinon.spy(),
+    };
+    const route = makeRoute({
+      pageConfig: {
+        pageKey: 'lastPage',
+        schema: {},
+        uiSchema: {},
+        errorMessages: {},
+        title: '',
+      },
+    });
+
+    const tree = SkinDeep.shallowRender(
```

Comment:
Ok, I hate to ask you to add another renderer to this file (we already have two!), but we are not using SkinDeep anymore and we would like to move from Enzyme to React Testing Library. So. I would prefer we not write new tests using either SkinDeep or Enzyme. If you insist on using Enzyme, I won't fight too hard, but I would definitely rather we not use SkinDeep.

Reply:
Ok, makes sense... I tend to copy/paste other tests in the file to speed up the process. Switching to a different platform is usually a bit more time consuming. I'll look into it